src/css/_slider.scss

```diff
@@ -0,0 +1,77 @@
+// uncomment to visualize slider mechanism
+// input {
+//   border: solid;
+// }
+
+.population-slider {
+  position: absolute;
+  z-index: 1100;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  top: 10em;
+  left: 1em;
+}
+.min-max-slider {
+  width: 360px;
+  //   text-align: center;
+  height: 2em;
+  display: flex;
+  justify-content: space-between;
+  // margin-bottom: 50px;
+  // top: 10em;
+  // left: 1rem;
+}
+.min-max-slider > label {
+  display: none;
+}
```

Comment:
This is better as a nested rule in the `.min-max-slider`, with `> label` as the selector. 

But also, is it necessary to set this? Why hide the label?